net/isc-dhcp/files/dhcpd.init

```diff
@@ -116,11 +116,11 @@ typeof() {
 }
 
 update() {
-	local lhs="$1" family="$2" type="$3"
-	shift 3
+	local lhs="$1" leasetime="$2" family="$3" type="$4"
+	shift 4
 
 	[ $dynamicdns -eq 1 ] && \
-		echo -e "$PREFIX" "$lhs $family $type $@\nsend" >> $dyn_file
+		echo -e "$PREFIX" "$lhs ${leasetime} $family $type $@\nsend" >> $dyn_file
```

Comment:
Why do we need curlies around `leasetime`?

Reply:
@jempatel ?